CMakeLists.txt

```
@@ -120,7 +126,14 @@ add_library(vcpkglib
     ${VCPKGLIB_BASE_INCLUDES}
     ${VCPKGLIB_INCLUDES}
 )
-target_include_directories(vcpkglib PUBLIC include)
+target_include_directories(vcpkglib PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
```

```suggestion
    $<BUILD_INTERFACE:include>
```

I don't believe that the `${CMAKE_CURRENT_SOURCE_DIR}` is necessary here?